utility/src/main/java/org/oppia/android/util/formattor/FileSizeConversionUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.util.formattor
+
+import android.content.Context
+import org.oppia.android.util.R
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Utility to calculate file size. */
+@Singleton
+class FileSizeConversionUtil @Inject constructor(
```

Comment:
I don't think this will work correctly. Using R requires special regeneration in Bazel at the moment for proper interoperability with Gradle. I think this conversion needs to stay in the app layer.